src/ducks/SANCharts/Chart/Signals/helpers.js

```diff
@@ -0,0 +1,120 @@
+import COLOR from '@santiment-network/ui/variables.scss'
+import { getTextWidth } from '@santiment-network/chart/utils'
+import { clearCtx } from '../utils'
+
+export const SIGNAL_BELOW = 'BELOW'
+export const SIGNAL_ABOVE = 'ABOVE'
+
+const HEIGHT = 22
+
+const MARGIN_BOTTOM = 5
+const BUBBLE_MARGIN_BOTTOM = HEIGHT + MARGIN_BOTTOM
+
+const PADDING_LEFT = 8
+const PADDINGS = PADDING_LEFT * 2
+
+const MARGIN_LEFT = 5
+const TEXT_LEFT_MARGIN = MARGIN_LEFT + PADDING_LEFT
```

Comment:
@DmitriVanGuard Can you write which text (ticks/labels/tooltips)

Reply:
It's for the text inside the rect: `Click to ....`
![image](https://user-images.githubusercontent.com/25135650/73074752-5dba6000-3ecb-11ea-9d2a-b287c01d2a51.png)